react-app/src/pages/WeeklyWeatherPage.tsx

```diff
@@ -15,7 +15,7 @@ const WeeklyWeatherPage: React.FC = () => {
           `https://api.openweathermap.org/data/2.5/onecall?lat=33.44&lon=-94.04&appid=${process.env.REACT_APP_API_KEY}&units=metric`
         );
 
-        //TODO: itt mit kéne
+        //TODO: az apinak kellene interface, és az lenne a type az itemnek
         const asd = response.data.daily.map((item: any) => {
```

Comment:
item : IWeeklyWeather ?

Reply:
The object stored in the response has so many properties that i did not created an interface to match it.

Reply:
You don't have to cover all of the properties available on the object, you can select only the ones you use. If I'm not wrong.